src/Cli/Commands/InitOptions.cs

```diff
@@ -113,6 +115,11 @@ public class InitOptions : Options
         [Option("graphql.enabled", Required = false, HelpText = "Enables GraphQL endpoint for all entities. Supported values: true, false.")]
         public CliBool GraphQLEnabled { get; }
 
+        // Since the rest.request-body-strict option does not have a default value, it is required to specify a value for this option if it is
+        // included in the init command.
+        [Option("rest.request-body-strict", Required = false, HelpText = "When set to true, does not allow extraneous fields in the request body for REST.")]
```

Comment:
nit: As per discussion, please update the HelpText so that the Default Value is visible to the end user even if we dont set it here.

Reply:
done